Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -601,6 +603,18 @@ Next, navigate to a new website in Safari. You should see traced function calls
  21324 ms     | -[NSURLRequest initWithURL:0x106388b00 cachePolicy:0x0 timeoutInterval:0x106388b80
 ```
 
+#### Native Libraries Tracing
+
+As discussed earlier in this chapter, iOS applications can also contain native code (C/C++ code) and it can be traced using the `frida-trace` CLI as well. For example, to trace calls for `open` function, following command can be used:
```

Comment:
```suggestion
As discussed earlier in this chapter, iOS applications can also contain native code (C/C++ code) and it can be traced using the `frida-trace` CLI as well. For example, you can trace calls to the `open` function by running the following command:
```